conf/application.conf

```diff
@@ -23,6 +23,7 @@ play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
 
 # Primary entry point for all HTTP requests on Play applications
 play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
+play.http.filters = "uk.gov.hmrc.payedesstub.filters.Filters"
```

Comment:
@rreddy80 I THINK we can do this in a different way. On the bootstrap-play readme file it states "If you have set play.http.filters to a custom filter, we recommend that you remove this, and use play.filters.enabled instead. You can append extra filters with play.filters.enabled += or remove filters with play.filters.disabled += ." - Link : https://github.com/hmrc/bootstrap-play#filters This may mean that we can also remove Filters class and the XSSProtectionFilter class. Might be worth a try?

Reply:
yeah makes sense. I have now added the built-in framework provided security headers